nbodykit/cosmology/cosmology.py

```diff
@@ -35,8 +37,10 @@ class Cosmology(object):
       inferred by the curvature condition.
     * Non-cosmological constant dark energy can be used by specifying the
       ``w0_fld``, ``wa_fld``, and/or ``Omega_fld`` values.
-    * The ``sigma8`` attribute can be set to the desired value, and the internal
-      value of ``A_s`` will be automatically adjusted.
+    * To pass in CLASS parameters that are not valid Python argument names, use
+      the dictionary /keyward arguments trick, e.g.
+      `Cosmology(..., **{'temperature contributions': 'y'})`
+    * ``Cosmology(**dict(c)) is not supposed to work; use ``Cosmology.from_dict(dict(c))``.
 
```

Comment:
Can you use double ticks for the Cosmology(..., **{'temperature contributions': 'y'}) line? And close the ticks on Cosmology(**dict(c)) so the docs look okay?